src/pal/inc/rt/rpc.h

```diff
@@ -19,7 +19,10 @@
 #define __RPC_USER
 #define __RPC_FAR
 
+#ifndef DECLSPEC_UUID
 #define DECLSPEC_UUID(x) __declspec(uuid(x))
+#endif
```

Comment:
Is `DECLSPEC_UUID` is already defined for gcc?

Reply:
I have a kludge solution for that now. Let's review the whole thing and try to make it better.